src/main/java/dev/codesupport/web/common/service/service/CrudOperations.java

```diff
@@ -93,20 +98,28 @@ void setupValidationBean() {
         validation = validationToReturn;
     }
 
+    void preGetEntities(List<E> entities) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/codesupport/website-backend/pullRequest?prid=6300789)